tap_zendesk/schemas/call_legs.json

```diff
@@ -2,105 +2,186 @@
         },
         "conference_from": {
-            "type": "integer",
+            "type": [
+                "null",
+                "integer"
+              ],              
             "description": "ID of agent who initiated the 3rd party conference"
         },
         "conference_time": {
-            "type": "integer",
+            "type": [
+                "null",
+                "integer"
+              ],              
             "description": "Time that agent spent in 3rd party conference (if call has multiple conferences we will sum the conference time)"
         },
         "conference_to": {
-            "type": "integer",
+            "type": [
+                "null",
+                "integer"
+              ],              
             "description": "ID of user or agent who joined the 3rd party conference"
         },
         "consultation_from": {
-            "type": "integer",
+            "type": [
+                "null",
+                "integer"
+              ],              
             "description": "ID of the agent who requested the consultation"
         },
         "consultation_time": {
-            "type": "integer",
+            "type": [
+                "null",
+                "integer"
+              ],              
             "description": "How long the agent consulted with another agent while the customer was on hold. null if there was no consultation"
         },
         "consultation_to": {
-            "type": "integer",
+            "type": [
+                "null",
+                "integer"
+              ],              
             "description": "The id of the agent who received and accepted the consultation request. null if there was no transfer"
         },
         "created_at": {
-            "type": "string",
-            "format": "date",
+            "type": [
+                "null",
+                "string"
+              ],
+              "format": "date-time",
             "description": "When the leg object was created"
         },
         "duration": {
-            "type": "integer",
+            "type": [
+                "null",
+                "integer"
+              ],              
             "description": "How long in seconds the leg existed"
         },
         "forwarded_to": {
-            "type": "string",
+            "type": [
+                "null",
+                "string"
+              ],
             "description": "Phone number the call was forwarded to. Example: '+1311123456789'. It can be null on customer leg"
         },
         "hold_time": {
-            "type": "integer",
+            "type": [
+                "null",
+                "integer"
+              ],              
             "description": "How long in seconds the leg existed"
         },
         "id": {
-            "type": "integer",
+            "type": [
+                "null",
+                "integer"
+              ],              
```

Comment:
The default primary key for a stream is `id`, so is this it? IIRC if so, this can't be null otherwise there'll be an error on target-postgres saying something along the lines of can't update pkey or something like that.

https://github.com/Pathlight/tap-zendesk/blob/bbf80f3799feec5461cb6dd245a37b80639c6bf1/tap_zendesk/streams.py#L16
https://github.com/Pathlight/tap-zendesk/blob/bbf80f3799feec5461cb6dd245a37b80639c6bf1/tap_zendesk/streams.py#L59

Reply:
i'll change this for the primary key